### PR TITLE
Add two more lingering_ config options

### PR DESCRIPTION
**Pull request by sbeaulie**
#### Pull Request (PR) description
Adding two more lingering_ config options
lingering_close
lingering_time
See https://nginx.org/en/docs/http/ngx_http_core_module.html#lingering_timeout

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Comment by genebean** [comment]
The two failures for CentOS 6 seem to be due to the CentOS 6 base repo going away (http://mirror.centos.org/centos-6/6/readme)

**Comment by genebean** [comment]
Once CI goes green on https://github.com/voxpupuli/puppet-nginx/pull/1436 this PR can be rebased and testing here _should_ also be good.

**Review by genebean — APPROVED** [review]

File: manifests/init.pp [hunk 1]
```diff
@@ -93,6 +93,8 @@
   $client_body_timeout                                       = '60s',
   $send_timeout                                              = '60s',
   $lingering_timeout                                         = '5s',
+  Optional[String] $lingering_close                          = undef,
```
Comment (bastelfreak) [inline manifests/init.pp:96]: can we enforce the string a bit more? maybe at least `Optional[String[1]]`?

Comment (sbeaulie) [reply manifests/init.pp]: done.
for lingering_close list the 3 valid options as enum
for lingering_time enforce a non empty string

Comment (genebean) [reply manifests/init.pp]: Good catch @bastelfreak - I totally missed that.

**Review by genebean — APPROVED** [review]

**Pull request merged**